magnetic/Generic/passwd.c

```diff
@@ -2,9 +2,21 @@
 #include <stdlib.h>
 #include <string.h>
 #include <ctype.h>
+#include <limits.h>
```

Comment:
Another utility program.  Changes should be reverted, or if not, use `stdint.h`.  Recommend reversion though, as we're only really maintaining Glk ports of interpreters and not the utilities.

In fact, for all these "non-Glk" things that slipped through, I think the right thing is to have absolutely zero changes to them, then in upstream, remove them, and re-merge upstream into master.   It makes upstream slightly messy, but who cares; it'll be another record of removing unnecessary files.